### DISCUSSION TITLE
Enabling 'dark' or other variants for filters - not working

### DISCUSSION THREAD

**Discussion by jasheal** [tailwindlabs/tailwindcss#6201] [Help]
It appears from the docs that this should be possible but I'm having trouble getting to to work. Is it a bug or just not supported?

Works fine:

`<Image
            src="/images/B.png"
            alt=""
            width="350"
            height="500"
            className="filter blur"
          />`

**Does not work:**

`
<Image
            src="/images/B.png"
            alt=""
            width="350"
            height="500"
            className="filter dark:invert"
          />
`

Config:

`
...
variants: {
    extend: {
      filter: ['dark', 'hover'],
    },
  },
`

**Comment by adamwathan** [comment]
Here's a working demo, see if this helps at all:

https://play.tailwindcss.com/Jy1huadZ2e

https://user-images.githubusercontent.com/4323180/143485718-48cf79fc-a7f1-4067-af30-08683f812266.mov
